apps/docs/src/app/core/component-docs/calendar/examples/calendar-mobile-example/calendar-mobile-example.component.html

```diff
@@ -55,8 +59,12 @@ <h3 fd-dialog-title>Pick a date</h3>
     </fd-dialog>
 </ng-template>
 
-<button fd-button label="Open Calendar in portrait mobile mode" (click)="openPortraitDialog(portraitDialog)"></button>
+<button fd-button
+        label="Open Calendar in portrait mobile mode"
+        style="margin-right: 10px"
+        (click)="openPortraitDialog(portraitDialog)">
+</button>
 
-<div>
-    <label fd-label>Date Picked: {{ datePicked.toDateString() }}</label>
-</div>
+<br><br>
```

Comment:
can we avoid using `br` in the examples? I think we have been doing it before but i find it more polluting - maybe we can add some margin/padding instead?